composer.json

```diff
@@ -19,12 +19,11 @@
     ],
     "require": {
         "php" : ">=5.5.0",
-        "ext-curl": "*"
+        "ext-curl": "*",
+        "ext-json": "*"
     },
     "require-dev": {
-        "phpunit/phpunit" : "~4.0",
-        "squizlabs/php_codesniffer": "2.*",
-        "codeception/codeception": "*"
+        "symfony/phpunit-bridge" : "*"
```

Comment:
Why not use phpunit itself?

Reply:
To test against php from 5.5 to 8.2

Reply:
Are there any plans to raise the minimum PHP version according to [EOL](https://www.php.net/supported-versions.php)?

Reply:
We'll see. If it's going hard to maintain legacy php version we'll rise requirements